packages/common_client/lib/src/ld_common_client.dart

```diff
@@ -179,7 +215,15 @@ final class LDCommonClient {
   ///
   /// If the return value is true, then the SDK has initialized, if false
   /// then the SDK has encountered an unrecoverable error.
-  Future<bool> start() {
+  ///
+  /// The [waitForNonCachedValues] parameters, when true, indicates that the SDK
+  /// will attempt to wait for values from LaunchDarkly instead of depending
+  /// on cached values. The cached values will still be loaded, but the future
+  /// returned by this function will not resolve. Generally this
+  /// option should NOT be used and instead flag changes should be listened to.
+  /// If [waitForNonCachedValues] is true, and an error is encountered, then
+  /// false may be returned even if cached values were loaded.
+  Future<bool> start({bool waitForNonCachedValues = false}) {
```

Comment:
So, if you want to use cached values on app start, but not on identify, then this supports that.

If you want to identify multiple contexts sequentially, and you want to ensure your cache gets updated, you can do that. You can also choose to use the cached values for the last context, being as that one would be updated in the background.

If you want to ensure you have "fresh" values when navigating to specific sections of your app, you could do that.

What this does not solve is the situation where someone wants to identify multiple contexts sequentially, wants the cache to be updated, but doesn't want the performance penalty. (One request was basically to allow the identify to continue and update the cache in the background, but concurrently start identifying a new context.)

Reply:
If you want flag values to "not change", then you can evaluate all relevant flags after the identify and use those values.

Alternatively, you could set the SDK to offline after completing the identify, and then only go online again when you want to specifically update values.